lib/services/metadata.test.js

```diff
@@ -155,4 +155,70 @@ describe(_.startCase(filename), function () {
         });
     });
   });
+
+  describe('checkProps', function () {
+    const fn = lib[this.title];
+
+    it ('returns false for a string prop with value false', function () {
```

Comment:
```suggestion
    it('returns false for a string prop with value false', function () {
```
Do this for every call of `it`